setup.cfg

```diff
@@ -11,7 +11,7 @@ search = __version__ = '{current_version}'
 replace = __version__ = '{new_version}'
 
 [bumpversion:file:docs/source/conf.py]
-search = version|release = {current_version}
+parse = release|version = "{current_version}"
```

Comment:
oh !  This `search` to `parse` needs to be backported to the cookiecutter I think.

Reply:
Yes, I opened an issue.